addons/mail/static/src/models/command/command.js

```diff
@@ -0,0 +1,30 @@
+odoo.define('mail/static/src/models/command/command.js', function (require) {
+'use strict';
+
+const { registerNewModel } = require('mail/static/src/model/model_core.js');
+const { attr } = require('mail/static/src/model/model_field.js');
+
+function factory(dependencies) {
+
+    class Command extends dependencies['mail.model'] {}
+
+    Command.fields = {
+        channel_types: attr(),
```

Comment:
Field not used, should filter the display of commands depending on this value. Not fixing it here.
task-2343850